packages/dev/src/main.ts

```diff
@@ -269,16 +274,14 @@ export class NetlifyDev {
       }
     }
 
-    const { pathname } = new URL(readRequest.url)
-    if (pathname.startsWith('/.netlify/images')) {
-      this.#logger.error(
-        `The Netlify Image CDN is currently only supported in the Netlify CLI. Run ${netlifyCommand('npx netlify dev')} to get started.`,
-      )
-
-      return
+    // 4. Check if the request matches an image.
+    const imageMatch = this.#imageHandler?.match(readRequest)
```

Comment:
I believe this should happen before the functions handler?

Reply:
I wasn't quite sure about where in order this should be. In initial attempt, I had it before functions, but after https://github.com/netlify/primitives/pull/255 was merged I assumed that was put in "correct" spot for image handling and just replaced the error with actual handling in place. 

I'll move it to be before functions